experimental/packages/otlp-exporter-base/src/transport/http-exporter-transport.ts

```diff
@@ -61,9 +51,33 @@ class HttpExporterTransport implements IExporterTransport {
       );
     });
   }
+
   shutdown() {
     // intentionally left empty, nothing to do.
   }
+
+  private _loadUtils(): Utils {
+    let utils = this._utils;
+
+    if (utils === null) {
+      // Lazy require to ensure that http/https is not required before instrumentations can wrap it.
+      const {
+        sendWithHttp,
+        createHttpAgent,
+        // eslint-disable-next-line @typescript-eslint/no-var-requires
+      } = require('./http-transport-utils');
```

Comment:
This should be updated to use dynamic import, but @pichlermarc is working on that elsewhere and needed to change the TS settings first. Conveniently, this is already used in an async context.